css/style.css

```
@@ -76,22 +76,89 @@ button {
 
 header {
   width: 1160px;
+  min-height: 881px;
   margin: 0 auto;
   display: flex;
   position: relative;
+  background-image: url("../img/light-back.png");
+  background-repeat: no-repeat;
+  background-position: center;
 }
 
 
 .header-logo {
   margin-right: 27px;
-  padding-top: 33px;
+  padding-top: 9px;
 }
 
 .header-nav {
   min-width: 350px;
-  padding-top: 33px;
   display: flex;
   justify-content: space-between;
+  align-items: start;
+  margin-top: 30px;
+}
+
+.header-nav__element {
+  display: inline-block;
+  position: relative;
+  height: 18px;
+  padding-bottom: 2px;
+  padding-left: 13px;
+  padding-right: 13px;
+}
+
+.header-nav__link {
+  color: #ffffff;
```

Такие значения можно сокращать до
```suggestion
  color: #fff;
```

---

ctrl+c ctr+v не руками писал поэтому так